README.md

```diff
@@ -49,3 +49,12 @@ $ nimble install zmq
 For more examples demonstrating many functionalities and patterns that ZMQ offers, see the ``tests/`` and ``examples/`` folder.
 
 The examples are commented to better understand how zmq works.
+
+
+### Log EAGAIN errno
+
+Sometimes EAGAIN error happens in ZMQ context; typically this is a non-ctritical error that can be ignored. Nonetheless, if you desire to logg or display such error you can compile with the flag ``-d:zmqEAGAIN`` and EAGAIN error will be logged using ``std/logging`` or ``echo`` to stdout if no logger handler is defined.
+
```

Comment:
How hard would it be to make this a runtime switch? Compile-time switches for libraries do not compose.

Reply:
Very easy. 

I will make the change.